more_autograding_examples/jupyter_notebook_autograding/config/config.json

```diff
@@ -0,0 +1,195 @@
+{
+  "autograding" : {
+        "submission_to_runner" : [  "*.ipynb", "*.png" ],
+        "work_to_details" : [ "**/*.ipynb", "**/*.png", "**/*.txt", "**/*.err" ]
+    },
+  "autograding_method": "docker",
+  "resource_limits" : {
+    "RLIMIT_NPROC" : 32
+  },
+  "container_options": {
+    "container_image": "submitty/jupyter:latest"
+  },
+  "testcases": [
+    {
+      "title": "Executes successfully",
+      "command": "python3 /grading_bin/jupyter_notebook_grader.py",
+      "points": 1,
+      "validation": [
+        {
+          "method": "fileExists",
+          "actual_file": "executed.ipynb",
```

Comment:
Then the instructor has control over the executed filename.
(This can be helpful if the instructor is modifying the students submitted file, modifying/replacing provided code, injecting hidden tests, etc.)